onnx/defs/schema.h

```diff
@@ -25,6 +25,8 @@
 #include "onnx/onnx-operators_pb.h"
 namespace ONNX_NAMESPACE {
 
+using FunctionBodyQueryFunction = std::function<bool(InferenceContext&)>;
```

Comment:
Hi, I think it will be better if we use a single "std::function<std::vector<NodeProto>(InferenceContext&)>" instead of a vector of pairs of (condition, body). In particular, this will be useful when we want to use the value of an attribute to create a specific ONNX node. Would it be possible to generalize this way?

Reply:
Sorry, I think some confusion happened in the HTML formatting with special characters. I mean a function that takes an input of type "InferenceContext&" and returns the body (a vector<NodeProto>).